.claude/PRINCIPLES.md

```
@@ -0,0 +1,677 @@
+
+**H3 (User Behavior)**: "Treatment group (visual menu) has ≥2x confidence rate vs. Control group (text-only)"
+
+**Rule**: Before implementing any feature, ask:
+- "Which hypothesis does this validate?"
+- "Can we validate H1-H3 without this feature?"
+- If answer is unclear → Don't build it (yet)
+
+### 2. Test-Driven Development (TDD)
+
+**Strict TDD Cycle** (Kent Beck):
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)